CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing Guidelines
+
+If you are in hurry, just please scan the **Quick Start** section.
+
+## Quick Start
+
+### Ideas, bugs...
+
+- File an [issue][issues].
+- [API Blueprint][] prepares direct support for testing and scenarios. Interested?
+  Check out [apiaryio/api-blueprint#21](https://github.com/apiaryio/api-blueprint/issues/21)!
+
+### Coding
+
+- Dredd is written in [CoffeeScript][].
+- There's [ready-made virtual machine][vde] to get you started very quickly.
+
+Recommended workflow:
+
+1. Fork Dredd, create a feature branch.
+2. Write tests, use `npm run tests:bdd`.
+3. Write code.
+4. Make sure [test coverage][] didn't drop.
+5. Send a Pull Request.
+
+## Handbook for Contributers and Maintainers
+
+### Maintainers
+
+[Apiary][] is the main author and maintainer of Dredd's [upstream repository][].
+Currently responsible people are:
+
+- [@netmilk](https://github.com/netmilk) - product decisions, feature requests
+- [@honzajavorek](https://github.com/honzajavorek) - lead of development
+
+### Programming Language
+
+Dredd is written in [CoffeeScript][] and is meant to be ran on server using
```

Comment:
This is duplicated with `- Dredd is written in [CoffeeScript][].` above in the "Coding" section, perhaps we can merge these sections together to prevent repetition?